### DISCUSSION TITLE
Interested parties should be individuals with affiliation listed

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/bestpractices#126] [Ideas]
Proposal: Switch from org or individual to individual with org affiliation listed.

This will better represent our community which consists of individual contributors. It encourages participation and contributions regardless of your organization affiliation. If needed we can still make decisions based on representation of a single org based on the listed org for each individual.

Reference:
- https://github.com/cncf/sig-app-delivery/blob/master/operator-wg/charter.md#interested-parties
- https://github.com/cncf/toc/blob/main/workinggroups/serverless.md#initial-interested-parties
- https://github.com/cncf/toc/blob/main/workinggroups/storage.md#initial-interested-parties
- GitOps WG has individuals with the company name.  See https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/edit#heading=h.665fh095a0wv
- same for the operator WG https://github.com/cncf/sig-app-delivery/blob/master/operator-wg/charter.md#interested-parties

**Comment by jeffsaelens** [comment]
List individuals and put the entity they are representing next to their name (even if that is self-representation). Confirm that what is written in the Charter meets the spirit of what is discussed here.

**Reply by xmulligan** [reply]
I actually don't think anything needs to be changed in the charter https://github.com/cncf/cnf-wg/blob/master/charter.md#elections. Relevant sections below 

"Individuals not associated with or employed by a company or organization are allowed one vote. Each company/organization, or related group of companies (regardless of the number of maintainers associated with or employed by that company/organization), will have a maximum of one-third representation in votes. If the results of an election result in greater than 1/3 representation, the lowest vote getters from any particular company will be removed until representation on the committee is less than one-third."

"Each organization listed in the interested parties document has one vote. Interested parties can be added at any time, but must be added at least one week before any election to have a vote. Any contributor from an organization may cast the vote for that organization. Each organization can cast one vote for a co-chair candidate in each of the communities (Kubernetes (K8s) Community, Service Provider (SP) Community, and CNF Developer (Dev) Community), three total votes. Each organization can vote for as many tech leads as they see fit."

**Reply by iawells** [reply]
Make it so.
